pages/movies/[movieId]/lists.tsx

```diff
@@ -33,7 +31,7 @@ export default function AddToList() {
     // @ts-ignore
     const userQuery = getUserByUsername(username);
 
-    const lists = ["liked", "favorites"];
+    const lists = ["liked", "watched"];
```

Comment:
currently, you are only looking at liked and watched lists but when there will be more lists in the future so shouldn't you look at which lists that user has?

Reply:
It's not that it's only looking at that. These are just default lists that are always present.

Reply:
The loop afterwards adds all the other lists a user has.

Reply:
Alright perfect thanks for the clarification